app/src/main/java/com/locoquest/app/ui/theme/Theme.kt

```diff
@@ -28,7 +28,7 @@ private val LightColorPalette = lightColors(
 )
 
 @Composable
-fun IndividualAssignment304832Theme(
+fun LocoQuestTheme(
```

Comment:
Probably auto generated as there is no theme usage yet. Renamed to be inline with naming conventions
https://kotlinlang.org/docs/coding-conventions.html#property-names

Reply:
Yes, good catch.